client/dashboard/components/overlay/style.scss

```diff
@@ -0,0 +1,19 @@
+.dashboard-overlay {
+	position: fixed;
+	inset: 0;
+	z-index: 9999;
```

Comment:
Not sure about this `z-index`. I had to add it so the header from the router doesn't get above it: 

https://github.com/user-attachments/assets/98910923-88e9-4507-a4c0-e72f606879ee